pttrack/models.py

```diff
@@ -318,6 +323,15 @@ def notes(self):
 
         return note_list
 
+    def workup_notes(self):
```

Comment:
The API convention I was trying to establish for the specific getters was `all_stuff` (see `all_phones` below). I'm not sure how rigidly I adhere to this, but maybe we should start actually taking code conventions more seriously that there's >1 person working on the project?